ui/app/AppLayouts/Profile/views/wallet/ManageTokensView.qml

```diff
@@ -128,10 +133,74 @@ ColumnLayout {
     }
 
     Component {
-        id: supportedTokensListPanel
-        SupportedTokenListsPanel {
-            sourcesOfTokensModel: root.sourcesOfTokensModel
-            tokensListModel: root.tokensListModel
+        id: advancedTab
+        ColumnLayout {
+            spacing: 0
+            StatusBaseText {
+                Layout.fillWidth: true
+                Layout.topMargin: 18
+                Layout.bottomMargin: 18
+                text: qsTr("Token lists")
+                color: Theme.palette.baseColor1
+            }
+            SupportedTokenListsPanel {
+                Layout.fillWidth: true
+                Layout.fillHeight: true
+                sourcesOfTokensModel: root.sourcesOfTokensModel
+                tokensListModel: root.tokensListModel
+            }
+            StatusBaseText {
+                Layout.fillWidth: true
+                Layout.topMargin: 40 + 18
+                Layout.bottomMargin: 26
+                text: qsTr("Asset settings")
+                color: Theme.palette.baseColor1
+            }
+            StatusDialogDivider {
+                Layout.fillWidth: true
+            }
+            StatusListItem {
+                Layout.fillWidth: true
+                title: qsTr("Show community assets when sending tokens")
+
+                components: [
+                    StatusSwitch {
+                        id: showCommunityAssetsSwitch
+                        checked: true // FIXME backend (#13118)
+                        onCheckedChanged: {
+                            // FIXME backend (#13118)
+                        }
+                    }
+                ]
+                onClicked: {
+                    showCommunityAssetsSwitch.checked = !showCommunityAssetsSwitch.checked
+                }
+            }
+            StatusDialogDivider {
+                Layout.fillWidth: true
+            }
+            StatusListItem {
+                Layout.fillWidth: true
+                title: qsTr("Don’t display assets with balance lower than")
+
+                components: [
+                    CurrencyAmountInput {
+                        enabled: displayThresholdSwitch.checked
+                        currencySymbol: SharedStores.RootStore.currencyStore.currentCurrency
+                        value: 0.10
```

Comment:
It could be a `d` readonly property!

Reply:
Could be if you insist :)

Reply:
Ah on a second thought, what exactly you meant here?